mcs/tools/xbuild/Makefile

```diff
@@ -100,9 +101,11 @@ install-web-targets:
 NUGET_BUILDTASKS_REPO_DIR=$(topdir)/../external/nuget-buildtasks
 
 install-nuget-targets:
-	$(MKINSTALLDIRS) $(DESTDIR)$(NUGET_BUILDTASKS_TARGETS_DIR)
+	$(MKINSTALLDIRS) $(DESTDIR)$(NUGET_BUILDTASKS_TARGETS_DIR) $(DESTDIR)$(MSBUILD_EXTENSIONS_15_DIR)
 	$(INSTALL_DATA) $(NUGET_BUILDTASKS_REPO_DIR)/src/Microsoft.NuGet.Build.Tasks/Microsoft.NuGet.targets $(DESTDIR)$(NUGET_BUILDTASKS_TARGETS_DIR)
 	$(INSTALL_DATA) $(NUGET_BUILDTASKS_REPO_DIR)/src/Microsoft.NuGet.Build.Tasks/Microsoft.NuGet.props $(DESTDIR)$(NUGET_BUILDTASKS_TARGETS_DIR)
+	$(INSTALL_DATA) $(NUGET_BUILDTASKS_REPO_DIR)/src/Microsoft.NuGet.Build.Tasks/ImportBeforeAfter/Microsoft.NuGet.ImportBefore.props $(DESTDIR)$(MSBUILD_EXTENSIONS_15_DIR)/Imports/Microsoft.Common.props/ImportBefore
+	$(INSTALL_DATA) $(NUGET_BUILDTASKS_REPO_DIR)/src/Microsoft.NuGet.Build.Tasks/ImportBeforeAfter/Microsoft.NuGet.ImportAfter.targets $(DESTDIR)$(MSBUILD_EXTENSIONS_15_DIR)/Microsoft.Common.targets/ImportAfter
```

Comment:
I already have these files in that folder in my 5.8.0.23, where did these come from?

Reply:
The first was because the `14.0/Imports` was copied to `15.0` in https://github.com/mono/msbuild/blob/xplat-master/install-mono-prefix.sh#L78 .
And the second was from the `NuGet.Build.Tasks` package (https://github.com/mono/msbuild/blob/xplat-master/nuget-support/tv/Microsoft.Common.targets/ImportAfter/Microsoft.NuGet.ImportAfter.targets).
But this file was subsequently removed from there. So, we are taking it from here!

Reply:
Makes sense! Let's keep @directhex in the loop in case this needs manual work in the Linux packaging.

Reply:
@directhex heads up, there are more changes coming with respect to packaging. So, don't get started on this yet :)